parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -2000,6 +2000,44 @@ int VariableBinaryTypePrecision() :
     { return precision; }
 }
 
+/**
+ * Parses LIKE ANY, LIKE ALL, and LIKE SOME statements, and appends operands
+ * to given list.
+ */
+void LikeAnyAllSome(List<Object> list, Span s) :
+{
+    final SqlOperator op;
+    boolean isLike = true;
```

Comment:
nit: isLike could be a little misleading since "NOT LIKE" is still a form of LIKE. Suggest changing this variable name to "not".